src/com/projectkorra/projectkorra/earthbending/EarthDome.java

```diff
@@ -81,6 +69,26 @@ private Block getAppropriateBlock(final Block block) {
 
 	@Override
 	public void progress() {
+		for (int i = 0; i < 2; i++) {
+			for (final Location check : this.getCircle(center, this.radius + i, 10)) {
+				Block b = check.getBlock();
```

Comment:
This variable could be named `currentBlock` or something to make it more readable.